app/controllers/response_controller.rb

```diff
@@ -14,12 +14,7 @@ def create
     content = question.options[params[:selected_options]]
     queue_content = question.question_queue_content(params[:selected_options])
     # Create response from question
-    responses_params = {
-      :question_id => question.id,
-      :content => queue_content[:content],
-      :quiz_id => @quiz.id,
-      :is_category => queue_content[:question_type] == 'categories'
-    }
+    responses_params = { :question_id => question.id, :content => queue_content[:content], :quiz_id => @quiz.id, :is_category => queue_content[:question_type] == 'categories' }
```

Comment:
Not sure if this was an improvement, the old format seems more readable, and the current line is very long.

Reply:
This was to address a maintainability issue of the method being too long !